src/Microsoft.DotNet.Build.Tasks/Targets/versioning.targets

```diff
@@ -0,0 +1,43 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="GetDailyBuildNumber" 
```

Comment:
At some point we will have a versioning.targets that will generate the AssemblyVersion.cs file. Do you see this information going into that file (I can see both directions)? If not then can you name it for its purpose, which is to generate the build number?